README.md

```diff
@@ -73,28 +73,59 @@ end
 ```
 
 ### Removing/Deleting an Attachment
-In order to allow the user to delete an attachment using the admin dashboard you need to do the following:
-1. create a controller action with a `delete` route
-2. point the `Field::ActiveStorage` field to that route
 
-here is an example (send the route name as a symbol):
-```ruby
-class ModelDashboard < Administrate::BaseDashboard
-  ATTRIBUTE_TYPES = {
-    attachment: Field::ActiveStorage.with_options({destroy_path: :custom_active_storage_destroy_path}),
-  }
-# ...
+`Administrate::Field::ActiveStorage` expects the presence of a route
+DELETE `/<namespace>/<resource>/:id/:attachment_name`, which will receive an optional parameter
+`attachment_id` in the case of an `ActiveStorage::Attached::Many`. For instance:
+
+```rb
+# routes.rb
+...
+namespace :admin do
+  ...
+  resources :users do
+    delete :avatars, on: :member, action: :destroy_avatar
+  end
+end
+
+# app/controllers/users_controller.rb
```

Comment:
shouldn't this be in the admin controller?

Reply:
OK fixed